src/recordlinker/models/mpi.py

```diff
@@ -121,14 +123,18 @@ class BlockingKey(enum.Enum):
     """
 
     BIRTHDATE = ("BIRTHDATE", 1, "Date of birth as YYYY-MM-DD")
-    SEX = ("SEX", 3, "Sex at birth; M, F or U")
+    SEX = ("SEX", 3, "Sex at birth; M or F")
     ZIP = ("ZIP", 4, "5 digital US Postal Code")
     FIRST_NAME = ("FIRST_NAME", 5, "First 4 characters of the first name")
     LAST_NAME = ("LAST_NAME", 6, "First 4 characters of the last name")
     ADDRESS = ("ADDRESS", 7, "First 4 characters of the address")
     PHONE = ("PHONE", 8, "Last 4 characters of the phone number")
     EMAIL = ("EMAIL", 9, "First 4 characters of the email address")
-    IDENTIFIER = ("IDENTIFIER", 10, "Identifier triplet with only last 4 character of the value. Format \"type:authority:value\"")
+    IDENTIFIER = (
+        "IDENTIFIER",
+        10,
+        "A colon separated string of the identifier type, first 2 characters of the authority and last 4 characters of the value",
```

Comment:
I'm wondering if authority refers to `Assigner` in [FHIR](https://www.hl7.org/fhir/R4/datatypes.html#Identifier) or is it sourced from somewhere else. If we're referring to `Identifier.assigner`, I think we should change the language to match.

Reply:
For FHIR documents, yes this is [pulled out of the Assigner resource](https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/hl7/fhir.py#L45).

There are a few places where Record Linker terms don't exactly align with FHIR specs (eg sex vs gender, first vs given, last vs family, etc).  In this case we used authority because that's what NBS uses in their system.  I'm open to the concept of changing features to align FHIR, but I'd want to have this discussion with the NBS mod team first.